src/main/webapp/app/home/home.component.html

```diff
@@ -1,44 +1,41 @@
-<div class="row">
-    <div class="col-md-3">
-        <span class="hipster img-fluid rounded"></span>
-    </div>
-
-    <div class="col-md-9">
-        <h1 class="display-4" jhiTranslate="home.title">Welcome, Java Hipster!</h1>
-
-        <p class="lead" jhiTranslate="home.subtitle">This is your homepage</p>
-
-        <div [ngSwitch]="isAuthenticated()">
-            <div class="alert alert-success" *ngSwitchCase="true">
-                <span id="home-logged-message" *ngIf="account" jhiTranslate="home.logged.message"
-                    [translateValues]="{ username: account.login }">You are logged in as user "{{ account.login }}".</span>
+<div [ngSwitch]="isAuthenticated()">
+    <div class="row justify-content-center ">
+        <div class="col-12 ">
+            <div class="entityWrapper home">
+                <h8 class="text-primary font-weight-bold" jhiTranslate="home.title">Welcome</h8>
+                <h8 class="text-primary" id="home-logged-message" *ngIf="account" jhiTranslate="home.title2"
+                    [translateValues]="{ username: account.login }">
+                    <span class="title-color"> {{ account.login }}!.</span></h8>
```

Comment:
Do we need to have a `<span>` inside a `h8`? Can't we just have the `{{ account.login }}!.` inside the `<h8>`?

Reply:
in this case we need to mantain this, because to the <h8> is refering to the first word (welcome: en; bem vindo(a): pt), and the <span> is refering to the {{ account.login }}!. 

In the first case we need a certain type of style (bold style) in the second case its not bold.